snc.sh

```diff
@@ -333,8 +359,13 @@ ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml controlPlane.architectu
 ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml baseDomain ${BASE_DOMAIN}
 ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml metadata.name ${CRC_VM_NAME}
 ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml compute[0].replicas 0
-${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml pullSecret "@HIDDEN_PULL_SECRET@"
-replace_pull_secret ${INSTALL_DIR}/install-config.yaml
+# if OKD install type was requested inject dummy pull-secret
+if [ ${OKD_INSTALL} == false ]; then
+    ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml pullSecret "@HIDDEN_PULL_SECRET@"
+    replace_pull_secret ${INSTALL_DIR}/install-config.yaml
+else
+    ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml pullSecret '{"auths":{"fake":{"auth": "bar"}}}'
+fi
```

Comment:
Might be more elegant (or too convoluted??) to do:
```
${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml pullSecret '{"auths":{"fake":{"auth": "bar"}}}'    
if [ ${OKD_INSTALL} == false ]; then
    replace_pull_secret ${INSTALL_DIR}/install-config.yaml
fi
```
and to change the @HIDDEN_PULL_SECRET@ pattern which is currently used in `replace_pull_secret`. Maybe `replace_pull_secret` could be always called, and have an early return when it's an okd install?